### PR TITLE
Fix dashboard UI bugs and more

### PR DESCRIPTION
**Pull request by CL-Jeremy**
This PR fixes a few UI bugs I spontaneously encountered:

- Fixes emojis in repo titles getting head-cut and tail-cut in dashboard feed due to introduction of 1.25 em emojis at 1 em line-height, by simply using the original 1 3/7 em value of `semantic.css`
- Fixes regression (too long repo names should be capped to 70%) in #13828 due to flex children not respecting properties like `overflow: hidden;`, and removes a block of dead style code
- Follow-up to #14761, removes extraneous code for top navbar and correct right margin for Font Awesome
- Fixes color emphasis inversion in arc-green theme for top right buttons (edit, delete) on commit view boxes

### CODE REVIEW

**Comment by CL-Jeremy** [comment]
Just came to the idea to use the `server` octicon for admin settings. Looks pretty understandable to me. Will add if people like it.

**Comment by lunny** [comment]
Could you give some screenshots?

**Comment by CL-Jeremy** [comment]
Snipaste_2021-02-22_06-22-33

File: web_src/less/_base.less [hunk 1]
```diff
@@ -363,9 +363,15 @@ a.muted:hover,
   background: var(--color-hover);
 }
 
-.ui.menu .ui.dropdown .menu > .selected.item {
-  color: var(--color-text) !important;
-  background: var(--color-hover) !important;
+.ui.menu .ui.dropdown .menu  {
```
Comment (CL-Jeremy) [inline web_src/less/_base.less]: `stylelint` is complaining about the extra space here. Will fix this later. Waiting for reactions regarding switching to  
 Site Administration

**Comment by CL-Jeremy** [comment]
@noerw seems to be confused about what I meant to replace with the server icon. It's this button here. Do you think it's a good replacement to the triple gear icon?

Currently in this PR:
Bildschirmfoto 2021-02-24 um 09 41 34

Proposal:
Bildschirmfoto 2021-02-24 um 09 35 27

**Comment by CL-Jeremy** [comment]
Pushing for now so I won't forget about it if PR takes too long. Will revert last commit if people object the change.

**Review by lafriks — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Pull request merged**